### PR TITLE
Add MAINTAINERS.adoc 

### PR DESCRIPTION
**Pull request by Vandit1604**
Add the guide to versioning the documentation. Currently named it `MAINTAINERS.adoc` it can be changed though.

### CODE REVIEW

File: MAINTAINERS.adoc [hunk 1]
```diff
@@ -0,0 +1,46 @@
+= How to Version Documentation
+
+== 1. Create Versioned Branch:
+
+- Begin by creating a dedicated branch for a new release, following semantic versioning, for example, _2.426.1_.
```
Comment (krisstern) [inline MAINTAINERS.adoc:5]:
```suggestion
- Begin by creating a dedicated branch for a new release, following semantic versioning, for example, _2.426.x_.
```

File: MAINTAINERS.adoc [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+== 2. Update `antora.yml` in Versioned Branches:
+
+- Ensure that all `antora.yml` files within this branch contain a consistent value for the `version` key. This value will be displayed to the users accessing the documentation.
+
+== 3. Add Branch to `antora-playbook.yml`:
+
+- Add the newly created branch, such as _**2.426.1**_, into the `antora-playbook.yml` file located in the playbook directory. This inclusion instructs Antora to fetch this branch and generate documentation for it, aligned with the version specified in the `antora.yml` file.
```
Comment (krisstern) [inline MAINTAINERS.adoc:13]:
```suggestion
- Add the newly created branch, such as _**2.426.x**_, into the `antora-playbook.yml` file located in the playbook directory. This inclusion instructs Antora to fetch this branch and generate documentation for it, aligned with the version specified in the `antora.yml` file.
```

File: MAINTAINERS.adoc [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      branches: [main,2.426.1]
+      start_paths: [docs/tutorials, docs/user-docs, docs/solutions]
+      # developer docs are un-versioned that's why they are to be fetched individually
+    - url: https://github.com/Vandit1604/jenkins-docs.git
+      branches: [main,2.426.1]
+      start_paths: [docs/dev-docs, docs/events, docs/security, docs/sigs, docs/projects, docs/images, docs/books, docs/community, docs/project, docs/about, docs/download]
+----
+
+
+- The `content.sources.branches` key should be updated with the name of the newly created _**2.426.1**_ branch
```
Comment (krisstern) [inline MAINTAINERS.adoc:29]:
```suggestion
- The `content.sources.branches` key should be updated with the name of the newly created _**2.426.x**_ branch
```

**Review by krisstern — CHANGES_REQUESTED** [review]

File: MAINTAINERS.adoc [hunk 4]
```diff
@@ -0,0 +1,55 @@
+= How to Version Documentation
+
+== 1. Create Versioned Branch:
+
+- Begin by creating a dedicated branch for a new release, following semantic versioning, for example, _2.426.x_ since software release aren't mapped to patch releases but to LTS stable releases.
+- This branch should be created 2 weeks prior the minor release when the release candidate is released.
```
Comment (krisstern) [inline MAINTAINERS.adoc:6]:
```suggestion
```

File: MAINTAINERS.adoc [hunk 5]
```diff
@@ -0,0 +1,55 @@
+= How to Version Documentation
+
+== 1. Create Versioned Branch:
+
+- Begin by creating a dedicated branch for a new release, following semantic versioning, for example, _2.426.x_ since software release aren't mapped to patch releases but to LTS stable releases.
+- This branch should be created 2 weeks prior the minor release when the release candidate is released. 
+ 
+NOTE: The main branch will contain documentation for the next release and when a release candidate will be released and another branch is created from the `main` branch. In simple terms, The `main` branch will always contains documentation for next release and 2 weeks before the release, we will create a branch.
```
Comment (krisstern) [inline MAINTAINERS.adoc:8]:
```suggestion
NOTE: The main branch will contain documentation for the next release and when a release candidate will be released and another branch is created from the `main` branch. In simple terms, The `main` branch will always contains documentation for the next release and some time before the release, we will create a versioned branch. 
```

**Review by krisstern — CHANGES_REQUESTED** [review]

**Comment by Vandit1604** [comment]
@krisstern I have added all the necessary information need to start versioning the documentation. Is there any question that jumps at you that should be addressed in the file. If not we can merge this.

**Review by krisstern — APPROVED** [review]

**Pull request merged**